src/parser/deserializer.ts

```diff
@@ -35,6 +35,8 @@ export interface DeserializeOptions {
   promoteBuffers?: boolean;
   /** when deserializing will promote BSON values to their Node.js closest equivalent types. */
   promoteValues?: boolean;
+  /** when deserializing will return UUID type */
```

Comment:
see other comment about adding to the docs.

Reply:
sounds good!